tests/containers/rootless_podman.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test rootless mode on podman.
+# - add a user on the /etc/subuid and /etc/subgid to allow automatically allocation subuid and subgid ranges.
+# - check uids allocated to user (inside the container are mapped on the host)
+# - give read access to the SUSE Customer Center credentials to call zypper from in the container
+#   proper way to add Access Control List is via `setfacl` which is not available so we just do
+#   `chmod` instead!! This grants the current user the required access rights
+# - Test rootless container:
+#   * container is launched with default root user
+#   * container is launched with existing user id
+#   * container is launched with keep-id of the user who run the container
+# - Restore /etc/zypp/credentials.d/ credentials
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use utils;
+use containers::common;
+use containers::container_images;
+use suse_container_urls 'get_suse_container_urls';
+use version_utils qw(get_os_release);
+
+sub run {
+    my ($self) = @_;
+    my ($image_names, $stable_names) = get_suse_container_urls();
+    my ($running_version, $sp, $host_distri) = get_os_release;
+    my $runtime = "podman";
+
+    install_podman_when_needed($host_distri);
+    allow_selected_insecure_registries(runtime => $runtime);
+    my $user = $testapi::username;
+    assert_script_run "cat /etc/subuid | grep bernhard || echo /etc/subuid has no uid range for $user";
```

Comment:
and another thing , not sure why you "cat'ing" file when `grep` can do everything without cat 
`grep -q bernhard /etc/subuid || echo /etc/subuid has no uid range for $user`

Reply:
no reason. let me update it